server/insights/src/main/java/org/infinispan/server/insights/logging/Log.java

```diff
@@ -46,8 +46,9 @@ public interface Log extends BasicLogger {
    @Message(value = "Error configuring Red Hat Insight client", id = 32006)
    CacheConfigurationException insightsConfigurationError();
 
+   @LogMessage(level = ERROR)
```

Comment:
I don't think this should have ERROR level given there's a backup mechanism. I'd say at most warning.

Reply:
Reduced to INFO